Podcast/EpisodesBySeriesIdEndpointRequest.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  EpisodesBySeriesIdEndpointRequest.swift
+//  Podcast
+//
+//  Created by Natasha Armbrust on 3/30/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+import SwiftyJSON
+
+class EpisodesBySeriesIdEndpointRequest: EndpointRequest {
+    
+    var seriesID: String
+    var offset: Int!
```

Comment:
doesn't need to be Int! since you initialize in init, same with max, they can both be Int